packages/dashboard/src/components/queryEditor/iotSiteWiseQueryEditor/footer/disableAdd.spec.tsx

```diff
@@ -84,7 +85,7 @@ describe(disableAdd, () => {
     ).toBeFalsy();
   });
 
-  it('should return false if KPI is selected and has NO assets already added', () => {
+  it('should return true if KPI is selected and more than 1 property selected', () => {
```

Comment:
The test name says more than 1 property selected, yet the properties object is empty?

Reply:
no matter what when we have more than 1 property selected, iif the widget is kpi/status, we disable